### PR TITLE
created user, restaurant, user_restaurant, and friends models

### PR DESCRIPTION
**Pull request by pgayee694**
Issue #1

### CODE REVIEW

**Comment by bbuffington1662** [comment]
Below is my comments. Sorry about the nitpicking!
- For the restraint table: please verify that largest name from the zomato api in case one is larger than 64 characters.
- Do the tables have auto-increment on the ids?
- For naming standards choose either UpperCamelCasing or like_this not Like_this.
- Change the name of the table User_Restaurants to have restaurant be singular.
- Please update the app/__init__.py so the comments do not contain "stuff".
- Do you need to specify Integer(32)/Integer(64)? I know you should in regular tables.
- In migrations/versions/7546857c977e_.py upgrade() the names of the tables are lowercase?
- Non of the columns should be nullable besides my location in the user table.
- Also I apologize, but I didn't think about this before. We should have a Location table with an id and a name. And then instead of a user having a location string column it will instead have an id to the location table.

**Comment by pgayee694** [comment]
- Zomato API docs don't give a size on the string returns, so hopefully 64 chars is more than enough
- All tables has auto-increment on the ids
- Functions will be `underscore_named` while objects should be `camelCase` apart from models with will follow whatever naming convention we choose (right now i just went with `underscore_named` due to familiarity
- Integer sizes don't need to be specified

**Comment by bbuffington1662** [comment]
- By naming standards I meant you named the table User_Restaurant. Make it either UserRestaurant or user_restaurant

File: .gitignore [hunk 1]
```diff
@@ -55,20 +55,20 @@ coverage.xml
 *.mo
 *.pot
 
-# Django stuff:
+# Django files:
```
Comment (bbuffington1662) [inline .gitignore:58]: Thank You!!!!!!!!!!!!!!

**Review by bbuffington1662 — APPROVED** [review]

**Review by tkorisko — APPROVED** [review]

**Review by cheddartot — APPROVED** [review]

**Pull request merged**